bands/templates/bands/w9.html

```diff
@@ -10,6 +10,7 @@
 
 <form method="post" action="w9" enctype="multipart/form-data">
 <input type="hidden" name="id" value="{{ band.id }}" />
```

Comment:
The way it's coded now, it almost certainly is making a new one each time, yes.

To further protect against this, you should add a unique constraint to the ``band_id`` field, since that will prevent more than one row from existing in the database for the same band.